### DISCUSSION TITLE
Elsa 2 Services configuration

### DISCUSSION THREAD

**Discussion by DNDevCode** [elsa-workflows/elsa-core#716] [Q&A] [answered]
Hi all, 

I have add Elsa.Persistence.EntityFramework.Core 2.0.0 but  elsa.AddEntityFrameworkStores<SqlServerContext> ... no longer available ? how can i configure entityframework with elsa 2 ? 

Thank you

**Comment by sfmskywalker** [comment, marked as answer]
Hi @DNDevCode , you can find an example here: https://github.com/elsa-workflows/elsa-core/blob/feature/elsa-2.0/src/samples/persistence/Elsa.Samples.Persistence.EntityFramework/Program.cs

**Reply by DNDevCode** [reply]
Perfect Thank you
